skietbaan_fe/src/App.js

```diff
@@ -1,46 +1,23 @@
-
-
 import React, { Component } from 'react'
 import { connect } from 'react-redux';
 import './App.scss'
 import { getUser } from './api/github'
 import {BrowserRouter, Route, Switch, Redirect} from 'react-router-dom'
```

Comment:
Inconsistent use of single and double quotes. I'd just go for single quotes all over.